Sources/StreamChatSwiftUI/ChatChannelList/MoreChannelActionsView.swift

```diff
@@ -91,6 +106,24 @@ public struct MoreChannelActionsView: View {
         .onTapGesture {
             onDismiss()
         }
+        .fullScreenCover(isPresented: $isPresented) {
+            if let fullScreenView = presentedView {
+                ZStack(alignment: .topTrailing) {
+                    fullScreenView
```

Comment:
We still need the nav bar view here, since it displays some channel information. We can put the close on the left side, similarly to the GalleryView.